app/src/main/res/layout/message_activity.xml

```diff
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<?xml version = "1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
ScrollView ? Why ? the potrait layout is already fixed

Reply:
ScrollView is because the whole layout needs to be scrolled....

Reply:
If you use a scroll view, it will become scrollable and you want to make it fixed

Reply:
Btw it's resolved already. You only need to make the landscape scrollable